apps/token/src/routes/governance/components/proposal-detail-header/proposal-header.spec.tsx

```diff
@@ -0,0 +1,238 @@
+import { render, screen } from '@testing-library/react';
+import { generateProposal } from '../../test-helpers/generate-proposals';
+import { ProposalHeader } from './proposal-header';
+import type { Proposals_proposals } from '../../proposals/__generated__/Proposals';
+
+const renderComponent = (proposal: Proposals_proposals) => (
+  <ProposalHeader proposal={proposal} />
+);
+
+it('New market', () => {
```

Comment:
chore (non-blocking): imho it's a lot nicer when tests are wrapped in a describe, noting what functionality / component are they testing, and the `it` field to be more descriptive of the expectation we're testing, something like:

```
describe('Proposal header', () => {
  it('renders new market', () => {...})
  it('updates market', () => {...})
  ...
})
```

Reply:
Yeah, good shout, will do.